desktop/onionshare/resources/locale/en.json

```diff
@@ -43,24 +43,24 @@
     "gui_tor_settings_window_title": "Tor Settings",
     "gui_autoconnect_description": "OnionShare relies on the Tor Network, run by thousands of volunteers around the world.",
-    "gui_enable_autoconnect_checkbox": "Connect to Tor automatically",
-    "gui_autoconnect_failed_to_connect_to_tor": "Failed to Connect to Tor",
-    "gui_autoconnect_trying_to_connect_to_tor": "Trying to Connect to Tor...",
-    "gui_autoconnect_connection_error_msg": "Are you connected to the internet?",
-    "gui_autoconnect_bridge_description": "It's also possible that your internet is being censored. You might be able to bypass this using a bridge.",
-    "gui_autoconnect_bridge_detect_automatic": "Automatically determine my country from my IP address for bridge settings",
-    "gui_autoconnect_bridge_detect_manual": "Manually select my country for bridge settings",
+    "gui_enable_autoconnect_checkbox": "Auto-connect to Tor",
+    "gui_autoconnect_failed_to_connect_to_tor": "Could not connect to Tor",
+    "gui_autoconnect_trying_to_connect_to_tor": "Connecting to Tor…",
+    "gui_autoconnect_connection_error_msg": "Make sure you are connected to the Internet.",
+    "gui_autoconnect_bridge_description": "You might be able to connect using a bridge if your Internet connection is censored.",
+    "gui_autoconnect_bridge_detect_automatic": "Determine my country from my IP address for bridge settings",
+    "gui_autoconnect_bridge_detect_manual": "Select my country for bridge settings manually",
     "gui_autoconnect_bridge_setting_options": "Bridge Settings",
     "gui_autoconnect_start": "Connect to Tor",
     "gui_autoconnect_configure": "Network Settings",
-    "gui_autoconnect_no_bridge": "Try again without bridge",
-    "gui_autoconnect_try_again_without_a_bridge": "Try again without a Bridge",
-    "gui_autoconnect_circumventing_censorship": "Trying to resolve connectivity issues",
-    "gui_autoconnect_circumventing_censorship_starting_circumvention": "Starting censorship circumvention process",
-    "gui_autoconnect_circumventing_censorship_starting_meek": "Starting Meek for domain-fronting",
-    "gui_autoconnect_circumventing_censorship_requesting_bridges": "Requesting bridges from the Tor Censorship Circumvention API",
-    "gui_autoconnect_circumventing_censorship_got_bridges": "Got bridges! Trying to reconnect to Tor",
-    "gui_autoconnect_could_not_connect_to_tor_api": "Could not connect to the Tor API. Make sure you are connected to the internet before trying again.",
+    "gui_autoconnect_no_bridge": "Try again without bridges",
+    "gui_autoconnect_try_again_without_a_bridge": "Try again without bridges",
+    "gui_autoconnect_circumventing_censorship": "Resolving connectivity issues…",
+    "gui_autoconnect_circumventing_censorship_starting_circumvention": "Circumventing censorship…",
+    "gui_autoconnect_circumventing_censorship_starting_meek": "Establishing Meek bridge for domain-fronting…",
```

Comment:
"domain-fronting" sounds a little cryptic.

Reply:
I would agree to keeping domain-fronting as long as the upstream uses it to be consistent.